tezster.js

```diff
@@ -1,467 +1,204 @@
-.action(async function(){
-    var args = process.argv.slice(3);
-    const tezsterManager = require('./tezster-manager');
-    if (args.length < 4) {
-        console.log(tezsterManager.outputInfo("Incorrect usage of deploy command \n Correct usage: - tezster deploy contract-label contract-absolute-path init-storage-value account"));
-        return;
-    }
-    await tezsterManager.loadTezsterConfig(); 
-
-    let result = await tezsterManager.deployContract(args[0], args[1], args[2], args[3]);
-    console.log(result);
-    console.log(tezsterManager.outputInfo(`If you're using ${TESTNET_NAME} node, use https://${TESTNET_NAME}.tzstats.com to check contract/transactions`));
+    .command('deploy')
+    .usage('<contract-label> <contract-absolute-path> <init-storage-value> <account>')
+    .description('Deploys a smart contract written in Michelson')
+    .action(function(){
+        tezstermanager.deployContract();
 });
 
 //*******calls contract written in Michelson*/
 program
-.command('call')
-.usage('<contract-name/address> <argument-value> <account>')
-.description('Calls a smart contract with given value provided in Michelson format')
-.action(async function(){
-    var args = process.argv.slice(3);
-    const tezsterManager = require('./tezster-manager');
-    if (args.length < 3) {
-        console.log(tezsterManager.outputInfo("Incorrect usage of call command \n Correct usage: - tezster call contract-name argument-value account"));
-        return;
-    }
-    await tezsterManager.loadTezsterConfig(); 
-    
-    let result = await tezsterManager.invokeContract(args[0], args[1], args[2]);
-    console.log(result);
-    console.log(tezsterManager.outputInfo(`If you're using ${TESTNET_NAME} node, use https://${TESTNET_NAME}.tzstats.com to check contract/transactions`));
+    .command('call')
+    .usage('<contract-name/address> <argument-value> <account>')
+    .description('Calls a smart contract with given value provided in Michelson format')
+    .action(function(){
+        tezstermanager.callContract();
 });
 
 //*******gets storage for a contract*/
 program
-.command('get-storage')
-.usage('<contract-name/address>')
-.description('Returns current storage for given smart contract')
-.action(async function(){
-    var args = process.argv.slice(3);
-    const tezsterManager = require('./tezster-manager');
-    if (args.length < 1) {
-        console.log(tezsterManager.outputInfo("Incorrect usage of get-storage command \n Correct usage: - tezster get-storage contract-name"));
-        return;
-    }
-    await tezsterManager.loadTezsterConfig(); 
-    
-    let result = await tezsterManager.getStorage(args[0]);
-    console.log(result);
+    .command('get-storage')
+    .usage('<contract-name/address>')
+    .description('Returns current storage for given smart contract')
+    .action(function(){
+        tezstermanager.getStorage();
 });
 
-/* Restores an testnet faucet account */
+//******* To Create an contract */
```

Comment:
why do you need extra '/' before block comment (/* ... */)? You can remove that from everywhere.